cvat/apps/annotation/validation.py

```diff
@@ -0,0 +1,214 @@
+import io
```

Comment:
this validation is task-specific. At some level, either in file name or method name, you should say that this only applies to spotter bbox validation.

Reply:
I'll rename the function when I add validation for new task.